documentation/slate/source/includes/apis.md

```diff
@@ -52,12 +43,16 @@ ressource id:
 }
 ```
 
-The coverage id is *"regions": ["fr-idf"]* so you can ask Navitia on
-accessible local mobility services:
+> in this example, the coverage id is "regions": ["fr-idf"] 
+so you can ask Navitia on accessible local mobility services:
```

Comment:
I know it's not in the scope of your PR, but I think it's better to say `available` and not  `accessible`